hack/kind-setup-loopback-devices.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+
+MULTI_ZONAL="false"
+CLUSTER_NAME=""
+IPFAMILY="ipv4"
+
+SUDO=""
+if [[ "$(id -u)" != "0" ]]; then
+  SUDO="sudo "
+fi
+
+parse_flags() {
+  while test $# -gt 0; do
+    case "$1" in
+    --cluster-name)
+      shift; CLUSTER_NAME="${1}"
+      ;;
+    --ip-family)
+      shift; IPFAMILY="${1}"
+      ;;
+    --multi-zonal)
+      MULTI_ZONAL=true
+      ;;
+    esac
+
+    shift
+  done
+}
+
+parse_flags "$@"
+
+
+
+LOOPBACK_IP_ADDRESSES=(172.18.255.1)
+if [[ "$IPFAMILY" == "ipv6" ]] || [[ "$IPFAMILY" == "dual" ]]; then
+  LOOPBACK_IP_ADDRESSES+=(::1)
+fi
+
+if [[ "$MULTI_ZONAL" == "true" ]]; then
+  LOOPBACK_IP_ADDRESSES+=(172.18.255.10 172.18.255.11 172.18.255.12)
+  if [[ "$IPFAMILY" == "ipv6" ]] || [[ "$IPFAMILY" == "dual" ]]; then
+    LOOPBACK_IP_ADDRESSES+=(::10 ::11 ::12)
+  fi
+fi
+
+if [[ "$CLUSTER_NAME" != "*local2*" ]] ; then
+  LOOPBACK_IP_ADDRESSES+=(172.18.255.22)
+  if [[ "$IPFAMILY" == "ipv6" ]] || [[ "$IPFAMILY" == "dual" ]]; then
+    LOOPBACK_IP_ADDRESSES+=(::22)
+  fi
+fi
+
+if [[ "$CLUSTER_NAME" == "gardener-operator-local" ]]; then
+  LOOPBACK_IP_ADDRESSES+=(172.18.255.3)
+  if [[ "$IPFAMILY" == "ipv6" ]] || [[ "$IPFAMILY" == "dual" ]]; then
+    LOOPBACK_IP_ADDRESSES+=(::3)
+  fi
+elif [[ "$CLUSTER_NAME" == "gardener-local2" || "$CLUSTER_NAME" == "gardener-local-multi-node2" ]]; then
+  LOOPBACK_IP_ADDRESSES+=(172.18.255.2)
+  if [[ "$IPFAMILY" == "ipv6" ]] || [[ "$IPFAMILY" == "dual" ]]; then
+    LOOPBACK_IP_ADDRESSES+=(::2)
+  fi
+fi
+
+
+if ! command -v ip &>/dev/null; then
```

Comment:
Nit: Usually, we do not leave several empty lines.

```suggestion
fi

if ! command -v ip &>/dev/null; then
```